Classes/Domain/Model/Translation.php

```diff
@@ -89,13 +93,17 @@ protected function exactMatchSearch(Search $search)
     protected function caseInSensitiveMatchSearch(Search $search)
     {
         $searchString = $search->getSearchString();
+        $return = false;
         if (strpos(strtolower($this->getTranslationTarget()), strtolower($searchString)) !== false) {
-            return true;
+            $return = true;
```

Comment:
why not let it return true here? And in the next if and in the net method?

Reply:
here it would be ok, but in the next if-clause we would need another if-clause inside, IMHO the method is more readable and consistent with the $return variable (have a look to the whole method)